corehq/apps/users/util.py

```diff
@@ -355,12 +355,21 @@ def log_user_change(by_domain, for_domain, couch_user, changed_by_user, changed_
     if action == UserModelAction.UPDATE and not fields_changed and not change_messages:
         raise ValueError("missing both 'fields_changed' and 'change_messages' argument for update.")
 
+    if changed_by_user == SYSTEM_USER_ID:
+        changed_by_id = SYSTEM_USER_ID
+        changed_by_repr = SYSTEM_USER_ID
+    else:
+        changed_by_id = changed_by_user.get_id
+        changed_by_repr = cached_user_id_to_username(changed_by_id)
```

Comment:
Any reason not to just get the username from the user object directly?
```suggestion
        changed_by_repr = changed_by_user.raw_username
```

Reply:
I don't see a reason not to.